src/bootstrap-table.js

```diff
@@ -952,7 +952,7 @@ class BootstrapTable {
     pageList = pageList.map(value => {
       if (typeof value === 'string') {
         return value.toLowerCase() === opts.formatAllRows().toLowerCase() ||
-          ['all', 'unlimited'].includes(value.toLowerCase())
+        ['all', 'unlimited'].includes(value.toLowerCase())
```

Comment:
This line does not need to be modified

Reply:
auto formatting of phpstorm :-(
I reverted this :)